pkgs/aws/session-manager-plugin/registry.yaml

```diff
@@ -0,0 +1,16 @@
+packages:
+  - type: http
+    repo_owner: aws
+    repo_name: session-manager-plugin
+    description: This plugin helps you to use the AWS Command Line Interface (AWS CLI) to start and end sessions to your managed instances Resources
+    version_source: github_tag
```

Comment:
Is this really necessary?
`version_source: github_tag` is useful if the package has GitHub Tags but doesn't have GitHub Releases.
But https://github.com/aws/session-manager-plugin has releases, so I don't think `version_source: github_tag` is necessary.
https://github.com/aws/session-manager-plugin/releases

Reply:
I see. Actually, I didn't understand this setting. I think the policy of using github_release is good.